crypto/fips/fips_post.c

```diff
@@ -70,10 +70,6 @@
 # include "crypto/rand.h"
 # include "fips_locl.h"
 
-char* FIPS_show_version(void)
-{
-    return "Rocky Linux 8 OpenSSL Cryptographic Module Version Rocky8.20240321";
```

Comment:
Do we just revert the commit that added these functions?

Reply:
Seems so. Sorry.